10.9-libcxx/gettext-tools.info

```diff
@@ -73,24 +75,28 @@ InfoTest: <<
 	TestConflicts: <<
 		fpc
 	<<
+	# Possible failures in French locale tests
 	TestScript: <<
-		cd gettext-tools; LC_ALL=C /usr/bin/make -w -k check || exit 2
+		#!/bin/sh -ev
+		cd gettext-tools
+		LC_ALL=C /usr/bin/make -w -k check || echo "Checking for expected failures in test suites"
+		grep ^FAIL */test-suite.log | egrep -v 'lang-perl-1|lang-perl-2' && exit 2
```

Comment:
Is there any chance that system-egrep (BSD) may behave differently from fink-egrep (GNU) ? If so, egrep here should be `/usr/bin/egrep`.

Reply:
I don't see any difference either in the manpages or my experience – the regex is of the most basic type, and `-v` is also one of the fundamental options.